6.X/6.9.0/docs/01-command_line_arguments.md

```diff
@@ -0,0 +1,332 @@
+
+- Sets the `kubernetesServerVersion` within values.
+
+`--helm-install`
+
+- The installer will extract the necessary files for an installation using the `helm` command only. By default it will create a directory `helm-install` in the directory where the installer is being executed. Content of the directory:
+
+  - `values.hi.yaml`: the complete values generated by the `installer`
+  - `values.hi.nats.yaml` and `values.hi.nats.global.yaml`: values for the rendering of NATSJS
+  - `charts`: the Helm charts that make up the Sysdig onprem stack
+
+`--helm-install-out-dir`
+
+- To use a custom directory to output the files generated by `--helm-install` instead of the default.
+
+### ArgoCD Generation
+
+We have introduced a way to generate ArgoCD apps definitions so that the sysdig stack can be installed using ArgoCD.
+
+At the moment we only take care of the generation of the files, the actual deploy of these files in ArgoCD is left to the user.
+
+`--argocd (boolean)`
+
+Generates files needed to deploy the sysdig stack on an ArgoCD installation. If the ArgoCD output directory exists, it will be deleted and recreated. NOTE: that using this flag will automatically also run the generation of charts that you would obtain with the `--helm-install` cli flag, this is because the argoCD generation is strictly tied with the helm-charts being produced in a certain way.
+
+`--argo-repo-url (string)`
+
+URL of repo that will contain ArgoCD files and helm charts, expected in the form `git@github.com:ORGANIZATION/SAMPLE-REPO.git`. (default `git@github.com:ORGANIZATION/SAMPLE-REPO.git`). This will be replaced within the ArgoCD apps definition files.
+
+`--argo-repo-rev (string)`
+
+Name of the branch of the repo to use. (default "main"). This will be replaced within the ArgoCD apps definition files.
+
+example of hierarchy:
+```
+git@github.com:ORGANIZATION/SAMPLE-REPO.git
+    |
+    '- argocd-projects/
+    |        '- sysdig/
+    |        |    '- argocd/
+    |        |    |    '- sysdig-root/
+    |        |    |    '- sysdig-common-config/
+    |        |    |    '- sysdigcloud-infra/
+    |        |    |         [...]
+    |        |    '- helm-install/
+    |        |    |    '- charts/
+    |        |    |          '- chart-1/
+    |        |    |          '- chart-2/
+    |        |    |          [...]
+```
+
+`--argo-git-apps-dir (string)`
+
+Relative path from the repo root that will contain the folder with ArgoCD apps definitions. (default "argocd"). This will be replaced within the ArgoCD apps definition files.
+If we look at the example above the correct value for this would be `argocd-projects/sysdig/argocd`.
+
+`--argo-git-charts-dir (string)`
+
+relative path from the repo root that will contain the folder with charts. (default "helm-install/charts"). This will be replaced within the ArgoCD apps definition files. If we look at the example above the correct value for this would be `argocd-projects/sysdig/helm-install/charts`.
```

Comment:
```suggestion
The relative path from the repo root that will contain the folder with charts. The default is `helm-install/charts`. This will be replaced within the ArgoCD apps definition files. If you consider the example above, the correct value for this would be `argocd-projects/sysdig/helm-install/charts`.
```